.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 		"quotes": [2, "double", { "avoidEscape": true }],
 		"import/no-named-as-default": 0,
 		"semi": [2, "always"],
-		"vue/html-indent": [1, "tab"]
+		"vue/html-indent": [1, "tab"],
+		"no-console": "off"
```

Comment:
I'd rather we just add an ignore to the relevant lines in the source file than ignoring this altogether. See: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

Reply:
"no-console" is turned off so I can debug the deploy preview. It won't be included in the pull request that will be merged into the dev and master.

Will close this pull request soon once the proper pull request is ready.